### DISCUSSION TITLE
Packaged Game not working

### DISCUSSION THREAD

**Discussion by OnceUponAMonster** [shun126/UE5-DungeonGeneratorDemo#52] [Q&A]
I am testing out the plugin and I have it working correctly in the editor however after I package the game in shipping config the level does not load.

I am using a template so I'm sure there could be something there possibly but for it to work in the editor but not packaged is throwing me for a loop.

Need help please

**Comment by shun126** [comment]
Please let us know the environment in which the problem is occurring.

UnrealEngine version:
Plugin Version:

If possible, what error message did you get when you packaged it?

**Reply by OnceUponAMonster** [reply]
I am not receiving any error messages.  It packages fine. It just does not load the map during packaged gameplay.

The map loads in from the main menu inside the editor but after packaging and playing the actual packaged game, the procedural level will not load.

UnrealEngine version: 5.4.4
Plugin Version: 5.4.0

**Comment by OnceUponAMonster** [comment]
Some additional information:

I was using this template from the marketplace: 
https://www.unrealengine.com/marketplace/en-US/product/survivors-roguelike-multiplayer-game-template

It was a free asset from awhile back. Might be worth looking into to see why it's not working after packaging.

I tried packaging a different template and it works in both the editor and after packaging & now I am about to tests thru steam.

**Comment by OnceUponAMonster** [comment]
Packaged game and using Steam to play across the internet.

The host and client are not syncing.

The Doors are synced but the floors and walls are not. Since the doors seem to be in the correct place I do not think that this is a seed issue between players??

How can I get this up and working?

**Reply by shun126** [reply]
It is difficult to determine whether the issue of the level not loading after packaging is due to the assets.

I tested multiplayer, and in my environment, mesh synchronization worked without any problems. It will take a bit more time to identify the issue.

**Reply by OnceUponAMonster** [reply]
Did you test it over the internet with steam?

**Comment by OnceUponAMonster** [comment]
![image](https://github.com/user-attachments/assets/d6d555a6-4735-42e9-923e-530fac86830f)

This is what we are dealing with.

The client is the left picture and host is the right.

Packaged game using steam subsystem.

How  should we set this up so it replicated correctly?

**Reply by shun126** [reply]
I'm going to test Steam matchmaking, so please wait a moment.

**Reply by OnceUponAMonster** [reply]
Any luck fixing this? I see an update came through in the marketplace

**Reply by shun126** [reply]
If possible, could you let me know the error message?

**Reply by shun126** [reply]
I tried integrating the Steam subsystem into the demo program, and I was able to package it. If I can check the error log, I might be able to consider a solution.
